### PR TITLE
Lecture 01-b – improvements

### PR DESCRIPTION
**Pull request by michaelgold**
The world is naturally hierarchical, thus our learning machines require multiple layers that build on each other compositionally. We define multilayer feedforward neural networks with combinations of linear and nonlinear modules using PyTorch. Then, we train these networks through stochastic gradient descent using backpropagation.

-- Second part of Lecture 1 (28 Jan 2019) by Erica Dominic and Hongyu (Florence) Lu. Edited by Michael Gold

### CODE REVIEW

**Comment by Atcold** [comment]
Thanks!
